### DISCUSSION TITLE
New Document editor on GrapesJS

### DISCUSSION THREAD

**Discussion by GoodPHP** [GrapesJS/grapesjs#3921] [Show and tell]
Hi, we want to show our last work of Document editor.

Video:
https://www.youtube.com/watch?v=1NqYlm7umJo

What do you think?

Service on GrapesJS: https://devfuture.pro/grapesjs-development/

**Comment by micker** [comment]
interesting !
the premade column mode is cool

**Reply by GoodPHP** [reply]
Thanks!

**Comment by ronaldohoch** [comment]
Nice!

**Reply by GoodPHP** [reply]
Thanks!
